integration-tests/test_client_options.py

```diff
@@ -17,3 +21,25 @@ def test_set_ansible_host_info(insights_client):
     ret = insights_client.run("--ansible-host=foo.example.com", check=False)
     assert "Could not update Ansible hostname" not in ret.stdout
     assert ret.returncode == 0
+
+
+def test_no_upload(insights_client):
+    """Verify no traceback is returned when running '--no-upload' command
+    and the archive is created and saved on the system
+    """
+    archive_saved = "Archive saved at"
+    upload_message = "Successfully uploaded report"
+    archive_directory = "/var/cache/insights-client"
```

Comment:
since this variable seems to be useful also in other tests (potentially), I'd make it as a top-level variable, eg:
```python
ARCHIVE_CACHE_DIRECTORY = "/var/cache/insights-client"
```

Reply:
done